src/main/flags.sh

```diff
@@ -4,19 +4,175 @@
 #
 set -eu
 
+OS="$1"
```

Comment:
can we add a short comment at the top here that's like "if you added a dependency that's causing a linking error, here's how to find out what library is needed (opam depext or similar), here's what to update below etc."

Reply:
also something about what to do if it's a language v opam package

Reply:
Done.